src/statefulmodelinstance.cpp

```diff
@@ -35,15 +35,15 @@ const std::set<std::string> StatefulModelInstance::SPECIAL_INPUT_NAMES{"sequence
 
 const Status StatefulModelInstance::extractSequenceId(const tensorflow::TensorProto& proto, uint64_t& sequenceId) {
     if (!proto.tensor_shape().dim_size()) {
-        SPDLOG_DEBUG("[Model: {} version: {}] Sequence id tensor proto does not contain tensor shape information", getName(), getVersion());
+        SPDLOG_DEBUG("Sequence id tensor proto does not contain tensor shape information");
```

Comment:
Why do we remove additional information which helps debugging?

Reply:
This cannot be method since it is used in RequestProcessor as a static function of StatefulModelInstance.

I considered passing here model name & version but I think that with logs you still have the thread id & could deduce from that the model name & model version.

Reply:
This is still a method of StatefulModelInstance class so we have access to `getName()` and `getVersion()`